### ISSUE TITLE
Finetuning is not stable

### ISSUE THREAD

**Issue by VimukthiRandika1997** [Peterande/D-FINE#169]
Finetuned the best model (object365+coco) on a custom dataset:

- Dataset size: 28, 000 (for training) and 5, 000 (for validation)
- No.of classes: 18
- No.of GPUs used: 4
- No.of epochs: 80
- Batch-size: 4

Observations:

- After 10 epochs, training loss were not reduced. it was slightly decreased

Have you guys have experienced this ? 
I would like to know your insights on this finetuning process!

Thank you!

**Comment by S130111** [comment]
I have the same problem, hope someone can solve it

**Comment by oneil512** [comment]
> For complex datasets, finetuning directly from Objects365 pre-trained weights tends to yield better results, and we recommend avoiding fine-tuning from the Obj365+COCO ckpt. Additionally, we observed that for some datasets with very few categories and simpler structures, training from scratch performs better.

What defines a simple structure?
